getting_started/in_depth_tutorials.rst

```diff
@@ -0,0 +1,25 @@
+.. _fundamental concepts:
+
+Fundamental concepts
+--------------------
+
+These tutorials walk step-by-step through the fundamental concepts of Lava introduced in :ref:`Lava Architecture <lava architecture>`. These tutorials tend to build on each other, therefore it's best to consume them in order.
+
+**Coming shortly:**
+
+* State monitoring:
+Lava offers probes to monitor the evolution of temporal state during execution. Learn how to create probes, retrieve timeseries and how to create custom visualization of network dynamics.
```

Comment:
See above

Reply:
I grouped all tutorials together and removed the "coming soon" parts.